.appveyor.yml

```diff
@@ -31,7 +31,8 @@ install:
   - conda info -a
   - conda install gtest cmake -c conda-forge
   - conda install xtl==0.4.1 -c QuantStack
-  - cmake -G "NMake Makefiles" -D CMAKE_INSTALL_PREFIX=%MINICONDA%\\LIBRARY -DBUILD_TESTS=ON -DDISABLE_VS2017=ON .
+  - conda install xsimd -c QUantStack
```

Comment:
typo :)

Reply:
good catch ;)

Reply:
Also, are we testing everything with xsimd enabled now? wouldn't it make sense to test some with XSIMD, and some without?

Reply:
ok, sorry, just looked at the results, and seems to make sense for now :)